script/release/notes/notes.js

```diff
@@ -564,15 +511,17 @@ const getNotes = async (fromRef, toRef, newVersion) => {
   return notes;
 };
 
+// FIXME: Chromium commit messages don't have this info anymore...
```

Comment:
What changed here?  Is it something we are differently in our roller PRs?

Reply:
User error. I was testing this from `master` but should've been in `9-x-y` :roll_eyes:

Reply:
Incorrect comment removed in f04d6a5